### DISCUSSION TITLE
Display blitzt auf

### DISCUSSION THREAD

**Discussion by accessburn** [pixelit-project/PixelIt#180] [Q&A]
Ich weiß mir nicht mehr zu helfen.
Das Display hat zwischendurch solche Flashs, also es Blitzt in hellen Farben einfach kurz auf, hat da evtl. eine Lösungsidee?

**Comment by foorschtbar** [comment]
Schon mal den MCU gewechselt?

**Comment by accessburn** [comment]
Du meinst einen anderen Wemos nehmen? Müsste ich einen weiteren besorgen. Finde ich nur merkwürdig, es funktioniert sonst alles, nur diese Flashs dazwischen nerven etwas :-p

**Reply by foorschtbar** [reply]
Würde erstmal mal den Wemos tauschen oder den Pin wechseln. Danach kann man weitersehen. Zusätzlicher ElKo, nen Widerstand an der Dataline etc..

**Reply by accessburn** [reply]
Okay, Widerstand an der Data ist ja nichts unübliches. 1k reicht? Was meinst du?
Denn die hab ich vorrätig. Aktuell sind mir nur die Wemos(e) ausgegangen.

**Comment by hamster65** [comment]
Du könntest mal versuchen, die Spannung leicht zu senken. Der ESP/wemos gibt ja 3,3V aus, die WS2812 wollen 5V. Wenn du die Spannung leicht reduzierst, z.B. auf 4-4,5V, verringerst du quasi den Abstand, weil die LEDs dann auch mit weniger Spannung laufen, während der ESPdurch den eingebauten Spannungsregler unverändert auf 3,3V bleibt.

Ansonsten würde ich auf Wackelkontakte an der Verbindung zu den WS2812 prüfen.

Ich kenne das Verhalten auch von Störungen durch Einstrahlungen eines Funkers. Lösung war, die Verbindung zu den WS2812 mit einer Diode auszustatten. Aber das war auch eine ungünstig lange Leitung.

Flackern tun die WS2812 auch, wenn sie kaputt sind. Alle LEDs hinter der defekten erhalten dann Datenmüll. Wenn du andere WS2812 hast, kannst du ja die mal dranhängen und schauen, ob die auch blitzen.

**Reply by accessburn** [reply]
Ist notiert, das mache ich dann wenn das mit dem Widerstand an der Datenleitung nix hilft. Ich hatte schon sowas ähnliches mal mit einem der 8x8er Displays. Wenn das nicht hilft geh ich mal deine Vorschläge ab. Danke auf alle Fälle.
